src/biocommons/seqrepo/seqrepo.py

```diff
@@ -94,14 +97,14 @@ class SeqRepo(object):
 
     def __init__(
         self,
-        root_dir,
-        writeable=False,
-        upcase=True,
-        translate_ncbi_namespace=None,
-        check_same_thread=False,
-        use_sequenceproxy=True,
-        fd_cache_size=0,
-    ):
+        root_dir: str,
+        writeable: bool = False,
+        upcase: bool = True,
+        translate_ncbi_namespace: Optional[str] = None,
+        check_same_thread: bool = False,
+        use_sequenceproxy: bool = True,
+        fd_cache_size: Optional[int] = 0,
```

Comment:
same comment as above
```suggestion
        fd_cache_size: int = 0,
```